### PR TITLE
support for running e2e with Catalog deployed from CSV

### PR DESCRIPTION
**Pull request by jboyd01**
support for deploying Service Catalog CSV and running upstream e2e tests

### CODE REVIEW

File: hack/create-subscription.sh [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+---
+apiVersion: operators.coreos.com/v1alpha2
+kind: OperatorGroup
+metadata:
+  name: service-catalog
+  namespace: kube-service-catalog
+spec:
+  selector:
+    matchLabels:
+      ns: kube-service-catalog
```
Comment (jpeeler) [inline hack/create-subscription.sh]: How does this label match the namespace created above? I'd expect to see something like `oc label ns kube-service-catalog ns=kube-service-catalog`.

Comment (jboyd01) [reply hack/create-subscription.sh]: I added a label to the kube-service-catalog namespace as suggested.  I can't tell you why it worked without it, but it did...  Anyway, I agree this looks more obvious now, and also validated it still works.

**Pull request merged**